### DISCUSSION TITLE
Thirdorder compile error

### DISCUSSION THREAD

**Discussion by tyjerome** [phoebe-team/phoebe#178] [Build help] [answered]
Dear Developers,

Hello, 

I was following the tutorial and met an error while trying to compile thirdorder to calculate anharmonic force constants.

bash compile.sh

error: command '/usr/bin/gcc' failed with exit code 1

Would you help me with this?

Thank you,
Yongjoong Shin.

**Comment by jcoulter12** [comment, marked as answer]
Hi Yongjoong, 

Is gcc available wherever you're trying to setup thirdorder.py? It just looks like you're missing some dependencies. 

If you're working somewhere that has the requisite dependencies to run Phoebe, you should also be able to compile thirdorder.py. Unfortunately, I don't see docs on the specific dependencies thirdorder.py requires, but it should at least require a working copy of spglib and gcc. 

Best, 
Jenny
